Source/WebCore/PAL/pal/spi/cocoa/CoreCryptoSPI.h

```diff
@@ -65,6 +65,17 @@ const struct ccdigest_info *ccsha512_di(void);
 extern const struct ccdigest_info ccsha512_ltc_di;
 
 void cccurve25519(ccec25519key out, const ccec25519secretkey sk, const ccec25519base);
```

Comment:
This function has also changed, why do we not need to update it too?

Reply:
right. I can add that too. Probably also need to check it's return value now and handle that.

Reply:
Have included other functions as well now and am now checking their return values.